### PR TITLE
Use glue instead of getOptlong

### PR DESCRIPTION
**Pull request by hideaki**
### Description
Use glue instead of getOptlong for parameter binding in SQL and mongo js query.

### Checklist

Make sure you have performed following items before submitting this pull request.
If not, please describe the reason.  

- [ ] Add test cases for this fix/enhancement
- [ ] Pass devtools::check()
- [ ] Pass devtools::test()
- [ ] Test installing from github
- [x] Tested with Exploratory

### CODE REVIEW

File: R/system.R [hunk 1]
```diff
@@ -215,14 +215,61 @@ getMongoURL <- function(host = NULL, port, database, username, pass, isSSL=FALSE
   return (url)
 }
 
+# glue transformer for mongo js query.
+# supports character, factor, logical, Date, POSIXct, POSIXlt, and numeric.
+# TODO: do we need to handle NA?
```
Comment (kei51e) [inline R/system.R]: How do you handle NA?

Comment (kei51e) [reply R/system.R]: can you add testcase for those cases?

File: R/system.R [hunk 2]
```diff
@@ -215,14 +215,64 @@ getMongoURL <- function(host = NULL, port, database, username, pass, isSSL=FALSE
...
+
+odbc_glue_transformer <- function(code, envir) {
+  val <- eval(parse(text = code), envir)
+  if (is.character(val) || is.factor(val)) {
+    # escape for SQL
+    # TODO: check if this makes sense for Dremio and Athena
+    val <- gsub("'", "''", val, fixed=TRUE) # both Oracle and SQL Server escapes single quote by doubling them.
+    val <- paste0("'", val, "'") # both Oracle and SQL Server quotes strings with single quote.
+  }
+  # TODO: How should we handle logical, Date, POSIXct, POSIXlt?
```
Comment (kei51e) [inline R/system.R]: I think we need NA handling here too.

File: R/system.R [hunk 3]
```diff
@@ -215,14 +215,64 @@ getMongoURL <- function(host = NULL, port, database, username, pass, isSSL=FALSE
...
+  #       Does expression like 1e+10 work?
+  glue::collapse(val, sep=", ")
+}
+
+bigquery_glue_transformer <- function(code, envir) {
+  val <- eval(parse(text = code), envir)
```
Comment (kei51e) [inline R/system.R]: I think we need NA handling here too.

File: R/system.R (hunk 2 above)
Comment (hideaki) [reply R/system.R]: Added TODO comment.

File: R/system.R (hunk 3 above)
Comment (hideaki) [reply R/system.R]: Added TODO comment.

**Pull request merged**